Example/TABTestKit_ExampleUITests/ExampleFeature.swift

```diff
@@ -22,7 +22,42 @@ class ExampleFeature: TestBase {
       Then(I: seeTheExampleScreen)
     }
   }
+}
+
+class BiometricTests: XCTestCase {
+  
+  private var app: XCUIApplication!
+  
+  private lazy var authenticateButton = app.buttons.element(matching: .button, identifier: "authenticateButton")
+  private lazy var authenticateLabel = app.staticTexts.element(matching: .any, identifier: "biometricStatusLabel")
+  
+  override func setUp() {
+    app = XCUIApplication()
+  }
+
+  override func tearDown() {
+    app = nil
+  }
   
+  func test_launchWithBiometricsDisabled() {
+    performUnenrollment()
+    app.launch()
+    Scenario("Launching the app with biometrics disabled") {
+      Given(I: seeBiometricsUnavailableStatus())
+      Given(I: seeAuthenticateButtonIsDisabled())
+    }
+  }
+  
+  func test_biometricsEnabled() {
+    performEnrollment()
+    app.launch()
```

Comment:
These should all be in step functions and called as Given/When/Thens inside scenarios.

`Given: biometricsAreEnrolled`
`When I: launchTheApp` 
etc